backend/baseclass.pm

```diff
@@ -568,6 +568,7 @@ sub reset_consoles {
     # we iterate through all consoles
     for my $console (keys %{$testapi::distri->{consoles}}) {
         #next if ($console eq 'x3270');
+        next if $self->console($console)->{args}->{skip_console_reset};
```

Comment:
as we are already in the console what about naming the attribute just `skip_reset`? Or maybe it's better to describe the nature of the console, not what should be done with, like, `persistent`?

Reply:
OK, `persistent` seems reasonable. Just thinking about documentation update. But existing http://open.qa/api/testapi/#_add_console is quite generic with `console parameters are console specific` without any other reference. Is it worth to mention that `persistent` is on of possible specific parameters?

Reply:
Updated `testapi.pm` doc section.

Reply:
Great that you found this place for documentation :) I did not ask you to add it as I also saw that `add_console` is just forwarding the hash of potential options